config/locales/en/course/forum/topics.yml

```diff
@@ -0,0 +1,19 @@
+en:
+  course:
+    forum:
+      topics:
+        topic:
+          started_by: 'Started by '
+          last_post: 'Last Post: '
```

Comment:
@allenwq @xzhflying in general localisation strings should not have trailing spaces or anything like that, they should be completely standalone.